students/jinee_han/lesson03/List_Lab.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python3
+
+# --------------------------------
+# 06/29/19 Jinee Han
+# Python Programming Lesson 3
+# List Lab
+# ---------------------------------
+
+# Series 1
+'''
+Create a list that contains “Apples”, “Pears”, “Oranges” and “Peaches”.
+Display the list (plain old print() is fine…).
+Ask the user for another fruit and add it to the end of the list.
+Display the list.
+Ask the user for a number and display the number back to the user and the fruit corresponding to that number (on a 1-is-first basis).
+Remember that Python uses zero-based indexing, so you will need to correct.
+Add another fruit to the beginning of the list using “+” and display the list.
+Add another fruit to the beginning of the list using insert() and display the list.
+Display all the fruits that begin with “P”, using a for loop.
+'''
+
+print ("Testing series 1.\n")
+fruit_list = ["Apples","Pears","Oranges","Peaches"] # Create a list
+print (fruit_list,'\n') # Print the list
+
+ask_user = input ("Would you like to add a fruit? (y/n)") # Ask the user
+if ask_user == "y": # Add the fruit to the list
+    add_fruit = input ("Which fruit would you like to add\n?")
+    fruit_list.append(add_fruit)
+    print (fruit_list) # Display the appended list
+else:
+    print ("\nNo item was added.\n")
+
+fruit_list = ["Apples","Pears","Oranges","Peaches"]
+ask_numeber = input("Enter the number to see the fruit from the list\n") #Ask user for a number to display
+display_fruit = print(fruit_list[int(ask_numeber)-1]) # Display the fruit
+
+add_another_fruit_with_plus = input("Let's add another fruit. Please type in.\n ") # Add another fruit.
+fruit_list = [add_another_fruit_with_plus] + fruit_list# Adding by using '+'
+print (fruit_list)
+
+add_another_fruit_with_insert = input("Final addition. What is the fruit name?\n") # Add another fruit.
+fruit_list. insert(0,add_another_fruit_with_insert) # Adding by using Insert
+print ("Here is the final fruit list.")
+print (fruit_list)
+
+
+# Display all the fruits that begins with 'P'
+
+print ("\nLet's find out fruits start with P\n")
+for items in fruit_list:
+    if items.startswith('P'):
+        print(items)
+    else:
+        continue
```

Comment:
no need for the `else` and `continue` statements.

Reply:
Thank you